src/service/evm.rs

```diff
@@ -55,6 +55,34 @@ use substrate_prometheus_endpoint::Registry;
 
 use super::{rpc, FullBackend, FullClient, HostFunctions, ParachainBlockImport};
 
+/// The ethereum-compatibility configuration used to run a node.
+#[derive(Clone, Copy, Debug, clap::Parser)]
+pub struct EthConfiguration {
+	/// Maximum number of logs in a query.
+	#[clap(long, default_value = "10000")]
+	pub max_past_logs: u32,
+
+	/// Maximum fee history cache size.
+	#[clap(long, default_value = "2048")]
+	pub fee_history_limit: u64,
+
+	#[clap(long)]
+	pub enable_dev_signer: bool,
```

Comment:
maybe default to `false` 

WDYT?

Reply:
`bool` in clap defaults to false implicitly